examples/vrf/src/handlers/arbiter.rs

```diff
@@ -143,7 +143,7 @@ impl<E: Clock> Arbiter<E> {
                                 };
                                 let payload = payload(round, &sender, &msg.commitment);
                                 if !C::verify(Some(ACK_NAMESPACE), &payload, public_key, &ack.signature) {
-                                    disqualify= true;
+                                    disqualify = true;
```

Comment:
If you remove all changes other than this one and make the change requested in `examples/bridge/src/bin/dealer.rs`, I'll merge!

I don't think the other changes are an improvement.